x-pack/plugins/xpack_main/server/lib/feature_registry/feature_registry.test.ts

```diff
@@ -119,12 +119,12 @@ describe('FeatureRegistry', () => {
           api: ['someApiEndpointTag', 'anotherEndpointTag'],
           ui: ['allowsFoo', 'showBar', 'showBaz'],
         },
-      },
+      } as any,
```

Comment:
__nit:__ actually this `as any` does nothing for us here since we have `const feature: Feature` already that should allow `foo`, so it'd make sense to either remove `: Feature` or `as any`.

Not related to this PR, `FeatureWithAllOrReadPrivileges` and `Feature` types are not as strict as I wanted it to be since we support empty object for `privileges` and hence `Feature` can be supplied to the method that expects `FeatureWithAllOrReadPrivileges` (`Record<string, *>` is compatible with `{}` and `{}` is compatible with `{all?: *, read?: *}`). But I believe we can tackle this later if we ever want tighten this.

Reply:
> **nit:** actually this `as any` does nothing for us here since we have `const feature: Feature` already that should allow `foo`, so it'd make sense to either remove `: Feature` or `as any`.

Great point.

> Not related to this PR, `FeatureWithAllOrReadPrivileges` and `Feature` types are not as strict as I wanted it to be since we support empty object for `privileges` and hence `Feature` can be supplied to the method that expects `FeatureWithAllOrReadPrivileges` (`Record<string, *>` is compatible with `{}` and `{}` is compatible with `{all?: *, read?: *}`). But I believe we can tackle this later if we ever want tighten this.

👍